.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "crates/mfem-cpp/mfem"]
 	path = crates/mfem-cpp/mfem
-	url = git@github.com:mkovaxx/mfem.git
+	url = git@github.com:mfem/mfem.git
```

Comment:
note: This change has the benefit that we no longer need to maintain a mirror. It also has the downside of being exposed to changes outside of our control.

Reply:
Well, a binding to an external library is to accept that we don't fully control the source.  Still we may want to submit issues & PR to MFEM.